cyber/tools/cyber_recorder/info.cc

```diff
@@ -77,22 +77,21 @@ bool Info::Display(const std::string& file) {
   std::cout << std::endl;
 
   // message_number
-  std::cout << std::setw(w) << "message_number:" << hdr.message_number()
+  std::cout << std::setw(w) << "message_number: " << hdr.message_number()
             << std::endl;
 
   // channel_number
-  std::cout << std::setw(w) << "channel_number:" << hdr.channel_number()
+  std::cout << std::setw(w) << "channel_number: " << hdr.channel_number()
             << std::endl;
 
   // read index section
   if (!file_reader.ReadIndex()) {
     AERROR << "read index section of the file fail. file: " << file;
     return false;
   }
-  Index idx = file_reader.GetIndex();
 
   // channel info
-  for (int i = 0; i < idx.indexes_size(); i++) {
+  for (int i = 0; i < file_reader.GetIndex().indexes_size(); ++i) {
```

Comment:
Might not a good change, as the GetIndex() will be called every time.

Reply:
agree with @xiaoxq

Reply:
Thanks for both of your comments! will fix it soon.